### PR TITLE
{bio}[GCC/5.4.0-2.26] SOAPdenovo2 vr240

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.3, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/d199ac52bccdf307eb0dd3253cd0b593 for a full test report.

File: easybuild/easyconfigs/s/SOAPdenovo2/SOAPdenovo2-r240-GCC-5.4.0-2.26.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-src-%(version)s.tgz']
+patches = ['SOAPdenovo2-fix-unistd-includes.patch']
+checksums = [
+    'b58fdb9b14766a122992d23dba5e91bd733c86e0062b432181aa5c1e7f052bb7',  # SOAPdenovo2-src-r240.tgz
+    '7d3c526eb4494f071a376a5b2826acad9048cbdddaf0ea2d6771dd3968497d00',  # SOAPdenovo2-fix-unistd-includes.patch
+]
+
+# parallel build is broken
+maxparallel = 1
```
Comment (boegel) [inline easybuild/easyconfigs/s/SOAPdenovo2/SOAPdenovo2-r240-GCC-5.4.0-2.26.eb]: @vanzod Did you re-check whether this is still needed?

If it's still relevant, just use `parallel =1` which hardcodes it to `1`. With `maxparallel` EasyBuild will still check the number of available cores first, and then do `max(N, 1)` which will of course be `1`, so better hardcode to `1` straight away...

Comment (vanzod) [reply easybuild/easyconfigs/s/SOAPdenovo2/SOAPdenovo2-r240-GCC-5.4.0-2.26.eb]: Yes, it still needs to be built serially.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.3, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/04af5d15f3b7e352f7990b10fe21d59d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0df5aa8ad546c6b77861129403367510 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node2605.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5e936544ed301fb85c868528fcf13642 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**